sr_tactile_sensor_controller/src/sr_biotac_tactile_sensor_publisher.cpp

```diff
@@ -40,6 +40,7 @@ void SrBiotacTactileSensorPublisher::init(const ros::Time& time)
   for (unsigned i = 0; i < sensors_->size(); i++)
   {
     biotac_realtime_pub_->msg_.tactiles[i].electrodes.resize(sensors_->at(i).biotac.electrodes.size());
+    biotac_realtime_pub_->msg_.tactiles[i].pac.reserve(270);
```

Comment:
Could we read this from tactile_sensors.hpp?
Otherwise maybe we could create a static variable here, rather than have a magic number.
Alternatively, not reserving would not be too bad, given that after 135 calls of `update()` the vector would reach its maximum size and would not need any further reallocation.